docs/releases/4.0.md

```diff
@@ -249,7 +249,7 @@ There are also many improvements to the documentation both under the hood and in
  * Resolve Layout issue with page explorer's inner header item on small device widths (Akash Kumar Sen)
  * Ensure that `BaseSiteSetting` / `BaseGenericSetting` objects can be pickled (Andy Babic)
  * Ensure `DocumentChooserBlock` can be deconstructed for migrations (Matt Westcott)
- * Resolve frontend console error and unintented console logging issues (Matt Westcott, Paarth Agarwal)
+ * Resolve frontend console error and unindented console logging issues (Matt Westcott, Paarth Agarwal)
```

Comment:
I think this is actually meant to be about intended/unintended console warnings, not about indenting for console warnings.

```suggestion
 * Resolve frontend console error and unintented console logging issues (Matt Westcott, Paarth Agarwal)
```